tools/package-tools/build-package.ts

```diff
@@ -81,19 +81,18 @@ export class BuildPackage {
 
   /** Compiles the TypeScript test source files for the package. */
   async compileTests() {
-    return compileEntryPoint(this, testsTsconfigName)
-      .then(() => addImportAsToAllMetadata(this));
+    return compileEntryPoint(this, testsTsconfigName);
   }
 
   /** Creates all bundles for the package and all associated entry points. */
   async createBundles() {
     await this.bundler.createBundles();
   }
 
-  /** Compiles TS into both ES2015 and ES5, then updates exports. */
+  /** Compiles TS into both ES2015 and ES2020, then updates exports. */
   private async _compileBothTargets(p = '') {
     return compileEntryPoint(this, buildTsconfigName, p)
-        .then(() => compileEntryPoint(this, buildTsconfigName, p, this.esm5OutputDir));
+        .then(() => compileEntryPoint(this, buildTsconfigName, p, this.esm2020OutputDir));
```

Comment:
Should that be `esm2015OutputDir` according to `compileEntryPoint`? I think ES2020 should be default, and then the sub-folder /second compilation is es2015.